src/components/ScoreByGameId.js

```diff
@@ -3,72 +3,70 @@ import Requests from "./Requests";
 import "../styles/scores.css";
 
 class ScoresByGameId extends React.Component {
-    constructor(props) {
-        super(props);
-        this.state = {
-            id: '',
-            scores: []
-        };
-    }
+  constructor(props) {
+    super(props);
+    this.state = {
+      id: "",
+      scores: []
+    };
+  }
 
-    async requestState(id) {
-        try{
-        const response = await Requests.get("/scores/" + id);
-        this.setState({id, scores: response});}
-        catch (e) {
-            console.log("Error in ScoreById: "+e);
-        }
+  async requestState(id) {
+    try {
+      const response = await Requests.get("/scores/" + id);
+      this.setState({ id, scores: response });
+    } catch (e) {
+      console.log("Error in ScoreById: " + e);
     }
+  }
 
-    async componentDidMount() {
-        const id = this.props.match.params.id;
-        this.requestState(id);
-    }
+  componentDidMount() {
+    const id = this.props.match.params.id;
+    this.requestState(id);
+  }
 
-    async componentDidUpdate(nextProps, prevState) {
-        const id = this.props.match.params.id;
-        if (id !== prevState.id) {
-            this.requestState(id);
-        }
+  componentDidUpdate(nextProps, prevState) {
+    const id = this.props.match.params.id;
+    if (id !== prevState.id) {
+      this.requestState(id);
     }
+  }
 
-    getScoresInList = () => {
-        const list = this.state.scores.map((score, index) => {
-            return (
-                    <tr className="scoreListItem">
-                        <td >{index+1}</td>
-                        <td>{score.Username}</td>
-                        <td>{score.Points}</td>
-                    </tr>
-            );
-        });
+  getScoresInList = () => {
+    const list = this.state.scores.map((score, index) => {
+      return (
+        <tr key={index} className="scoreListItem">
```

Comment:
two things here: 1 you don't need const list just to return it afterwards you can just do return this.state.scores.....
2) Same for inside the map you can just do ````this.state.scores.map((score, index) => (
        <tr key={index} className="scoreListItem">